app/models/supplejack_api/user_set.rb

```diff
@@ -16,7 +16,7 @@ class UserSet
 
     field :name,             type: String
     field :description,      type: String,   default: ''
-    field :copyright,        type: Integer,  default: 2
```

Comment:
This old change was a mistake. Copyright was fine we needed to change the privacy.

Reply:
Bit confused by your comment - Is it a change you want to keep?